### ISSUE TITLE
maxpool in dynamic routing

### ISSUE THREAD

**Issue by ierosodin** [ImMrMa/SegCaps-pytorch#3]
I' wondering why there is a maxpool operator in dynamic part?

**Comment by MikeManifold** [comment]
Maxpool is just to implement some of the functions of dynamic routing in SegCaps.Not the maxpool in the net.
In SegCaps,dynamic routing is a little different from the CapNet.
Respect!

**Comment by ierosodin** [comment]
Thanks! I get the point. It's a part of softmax when implementing it without calling function.
